package.json

```diff
@@ -27,12 +28,15 @@
     "eslint": "^6.7.2",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-vue": "^7.0.0",
+    "husky": "^8.0.1",
```

Comment:
Shouldn't husky and lint-staged be a dev-dependency?

Reply:
They are in devDependency only 🤔

Reply:
My bad